keps/sig-network/3705-cloud-node-ips/kep.yaml

```diff
@@ -16,19 +16,18 @@ reviewers:
   - "@thockin"
 approvers:
   - "@thockin"
-  - TBD from sig-cloud-provider
 
 see-also:
   - "https://github.com/kubernetes/enhancements/issues/1664"
   - "https://github.com/kubernetes/enhancements/pull/1665"
 
 # The target maturity stage in the current dev cycle for this KEP.
-stage: alpha
+stage: beta
```

Comment:
Please update
https://github.com/kubernetes/enhancements/blob/master/keps/prod-readiness/sig-network/3705.yaml

[assign to me]